pyensembl/gtf.py

```diff
@@ -144,7 +143,7 @@ def dataframe(self, contig=None, feature=None, strand=None):
         if strand:
             strand = normalize_strand(strand)
 
-        if not isinstance(feature, (NoneType, str, unicode)):
+        if feature is not None and not isinstance(feature, str):
```

Comment:
I guess we're bailing on the possibility of using a unicode string in Python 2.7.x?